config/default.json

```diff
@@ -75,6 +75,7 @@
     "awsRegion": "us-east-1",
     "sqsQueueUrl": "",
     "pollingIntervalMS": 10000,
+    "maxAttemptsToRetrieveMessages": 3,
     "maxTimeToHoldMessageSec": 10800,
     "waitTimeForMessageSec": 30
```

Comment:
Should we set this to 0?

The wait time is usually meant to allow messages to collect so that SQS batches can be fuller and fewer API calls are needed overall.

A worker only picks up one job at a time (so it doesn't care about waiting for fuller SQS batches), and it's only started when there is >= 1 job to process (so we know it'll almost always have work to do).